src/main/javascript/directives/cat-select.js

```diff
@@ -89,16 +89,18 @@ function CatSelectController($scope, $log, catApiService, catSelectConfigService
             results: function (data, page) {
                 var more = (page * (options.size || 100)) < data.totalCount;
                 return {
-                    results: _.filter(data.elements, filterFunc),
+                    results: _.map(_.filter(data.elements, filterFunc), function (e) {
+                        return Object.assign({}, e, {name: _.escape(e.name)});
```

Comment:
is their a reason you modify the results? you escape during `formatResult` and `formatSelection` so this looks redundant